catroid/src/main/java/org/catrobat/catroid/io/ProjectAndSceneScreenshotLoader.java

```diff
@@ -192,6 +167,44 @@ public void run() {
 				}
 			});
 		}
+
+		File getScreenshotFile() {
```

Comment:
If you're motivated, please check the sanity of this code, and maybe come up with a better solution (if applicable)